static_src/actions/domain_actions.js

```diff
@@ -0,0 +1,24 @@
+
+/*
+ * Actions for domain entities. Any actions such as fetching, creating,
+ * etc should go here.
+ */
+
+import AppDispatcher from '../dispatcher.js';
+import { domainActionTypes } from '../constants';
+
+export default {
+  fetch(domainGuid) {
+    AppDispatcher.handleViewAction({
+      type: domainActionTypes.DOMAIN_FETCH,
+      domainGuid
+    });
+  },
+
+  receivedDomain(domain) {
+    AppDispatcher.handleServerAction({
+      type: domainActionTypes.DOMAIN_RECEIVED,
+      domain
+    });
+  }
+};
```

Comment:
@msecret what do you think about consolidating these routeActions and domainActions into routeActions?

Here's my thinking: its kind of some CF specific terminology to know the difference between a route and a domain since most of the time you'll actually want both to determine the URL for an application. By having them handled in the same store, but using different action creators and action types it makes it a bit more confusing to hold the difference between a route and a domain in your head.

I think it would make sense to bring all of this into the `route*.js` files. Thoughts?